conductor/lib/exceptions.py

```diff
@@ -0,0 +1,11 @@
+'''
+exceptions module
+
+This module handles all custom exceptions for the Conductor Client Tools
+All exceptions should at minimum inherit off of the Exception base class
+'''
+
+class UserCanceledError(Exception):
+    '''
+    Custom Exception to indicate that the user cancelled their action
```

Comment:
`s/cancelled/canceled/` https://www.grammarly.com/blog/canceled-vs-cancelled/

Reply:
ha, that's my bad.  When I wrote this class originally, I wasn't sure of the spelling (My gut said it was with two Ls, but my smaller gut said it was one).  So I looked it up online.  Which said that both are used!  But since we're an American company, I went with the american spelling.  But failed to change it everywhere.
I bet there is some good/interesting reading/opinions on how teams of international developers handle language differences/conventions when contributing to the same code base.

Reply:
Ha, I researched this very thing too - as I usually spell it "cancelled" - which I suppose validates my Scot/English heritage.   I went with the goodl ol' USA standard after reading up on it.